infrastructure/main.tf

```diff
@@ -154,6 +154,12 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = "${module.db.postgresql_database}"
 }
 
+resource "azurerm_key_vault_secret" "APP-INSTRUMENTATION-KEY" {
+  key_vault_id = "${module.send-letter-key-vault.key_vault_id}"
+  name         = "${var.component}-APP-INSTRUMENTATION-KEY"
```

Comment:
"app-insights-instrumentation-key" for name please

Reply:
in this app component name used as prefix in vars.